### ISSUE TITLE
Units: `Wyrm` ID -> `Frostfire Monarch` 

### ISSUE THREAD

**Issue by knyghtmare** [knyghtmare/War_of_Legends#179]
### Describe the desired feature

- Just another instance of devs breaking add-ons by adding a unit with ID that already exists in this add-on. They should get hexed by witches at this point...

- WoL-using Add-ons affected: 
  - Journey of a Frost Mage
  - Legends Conquest
  - Struggle for Freedom
  - Need to check if others are affected as well

**Comment by knyghtmare** [comment]
The breaking PR: https://github.com/wesnoth/wesnoth/pull/7859

**Comment by babaissarkar** [comment]
I'd suggest not using "Wyrm Dragon", since Wyrm and Dragon have very similar meanings. So far in Wesnoth I saw that Wyrms mean two legged reptiles with two wings and two feet, while Dragons have four feet and two wings.

Even worse, I did some quick research and found this:
https://www.christophercant.com/articles/dragons-wyverns-drakes-wyrms

What we are referring to as "Wyrm" should actually be "Wyvern".

![Draconic Creatures](https://i.pinimg.com/736x/6d/8e/f9/6d8ef994886041eb3e6b6c732d959612.jpg)
